test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -257,3 +257,74 @@ def test_execjob_end_delete_request(self):
                           interval=2)
             msg = "Got expected log_msg on host:%s" % host
             self.logger.info(msg)
+
+    def test_execjob_end_reject_request(self):
+        """
+        Test to make sure hook job reject message should appear in mom log
+        in case sister mom went down before executing execjob_end hook
+        """
+
+        if len(self.moms) != 2:
+            self.skip_test(reason="need 2 mom hosts: -p moms=<m1>:<m2>")
```

Comment:
you do not need exact 2 hosts but at least 2 hosts. so you should check < 2 then skip.